docker-compose.yml

```diff
@@ -27,6 +27,14 @@ services:
       - node_db:/home/epoch/node/data/mnesia
       - node_keys:/home/epoch/node/keys
 
+  proxy:
```

Comment:
Do we actually need this?
As of [0.24.0](https://github.com/aeternity/epoch/blob/v0.24.0/docs/release-notes/RELEASE-NOTES-0.24.0.md) CORS are included in Epoch.